tests/test_data_sim.py

```diff
@@ -33,10 +35,20 @@
 MODE_SOLVER = make_mode_solver_data()
 FLUX = make_flux_data()
 FLUX_TIME = make_flux_time_data()
+DIFFRACTION = make_diffraction_data()
 
 # for constructing SimulationData
-MONITOR_DATA = (FIELD, FIELD_TIME, MODE_SOLVER, PERMITTIVITY, MODE, FLUX, FLUX_TIME)
-MONITOR_DATA_SYM = (FIELD_SYM, FIELD_TIME_SYM, MODE_SOLVER, PERMITTIVITY_SYM, MODE, FLUX, FLUX_TIME)
+MONITOR_DATA = (FIELD, FIELD_TIME, MODE_SOLVER, PERMITTIVITY, MODE, FLUX, FLUX_TIME, DIFFRACTION)
+MONITOR_DATA_SYM = (
+    FIELD_SYM,
+    FIELD_TIME_SYM,
+    MODE_SOLVER,
+    PERMITTIVITY_SYM,
+    MODE,
+    FLUX,
+    FLUX_TIME,
+    DIFFRACTION,
```

Comment:
Could you add a diffraction monitor to SIM_ALL in tests/utils.py? just to make sure that it passes all of the other components tests that are checking with SIM_ALL?